<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UltragridWindow</class>
 <widget class="QMainWindow" name="UltragridWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UltraGrid</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6" stretch="6,0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSplitter" name="previewsSplitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="PreviewWidget" name="capturePreview" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="previewLabel">
            <property name="text">
             <string>Local</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="PreviewWidget" name="displayPreview" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Remote</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="VuMeterWidget" name="vuMeter" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,1">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="title">
             <string>Video Source</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="videoSourceLabel">
               <property name="text">
                <string>Source</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="videoModeLabel">
               <property name="text">
                <string>Mode</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="videoModeComboBox"/>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="videoSourceComboBox">
               <property name="minimumContentsLength">
                <number>15</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Video Display</string>
            </property>
            <layout class="QFormLayout" name="formLayout_3">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Display</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="videoDisplayComboBox">
               <property name="minimumContentsLength">
                <number>15</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="previewCheckBox">
               <property name="text">
                <string>Preview</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
            <zorder>videoDisplayComboBox</zorder>
            <zorder>label_3</zorder>
            <zorder>previewCheckBox</zorder>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Video Compression</string>
            </property>
            <layout class="QFormLayout" name="formLayout_5">
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Compression</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="videoCompressionComboBox">
               <property name="minimumContentsLength">
                <number>15</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="videoBitrateLabel">
               <property name="text">
                <string>Bitrate</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="videoBitrateEdit"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Audio Capture</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Source</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="audioSourceComboBox">
               <property name="minimumContentsLength">
                <number>15</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Channels</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="audioChannelsSpinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>32</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Audio Playback</string>
            </property>
            <layout class="QFormLayout" name="formLayout_4">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Destination</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="audioPlaybackComboBox">
               <property name="minimumContentsLength">
                <number>15</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Audio Compression</string>
            </property>
            <layout class="QFormLayout" name="formLayout_6">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Compression</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="audioCompressionComboBox">
               <property name="minimumContentsLength">
                <number>15</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="audioBitrateLabel">
               <property name="text">
                <string>Bitrate</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="audioBitrateEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Network</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QFormLayout" name="formLayout_7">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Destination</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="networkDestinationEdit">
             <property name="text">
              <string extracomment="Target IP or host"/>
             </property>
             <property name="placeholderText">
              <string>localhost</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="portLabel">
             <property name="text">
              <string>Port</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="portLineEdit">
             <property name="placeholderText">
              <string>5004</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="fECLabel">
             <property name="text">
              <string>FEC</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="topMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="QCheckBox" name="fECCheckBox"/>
             </item>
             <item>
              <widget class="QToolButton" name="fecToolBtn">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QCheckBox" name="editCheckBox">
        <property name="text">
         <string>Edit arguments</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="arguments">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1062</width>
     <height>24</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <widget class="QMenu" name="menu_UltraGrid">
    <property name="title">
     <string>&amp;UltraGrid</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionShow_Terminal"/>
    <addaction name="actionVuMeter"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_UltraGrid"/>
    <addaction name="actionTest"/>
   </widget>
   <addaction name="menu_UltraGrid"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout_UltraGrid">
   <property name="text">
    <string>About UltraGrid</string>
   </property>
  </action>
  <action name="actionShow_Terminal">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show &amp;Log</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionAdvanced">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Advanced</string>
   </property>
  </action>
  <action name="actionUse_hw_acceleration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use hw. acceleration</string>
   </property>
  </action>
  <action name="actionVuMeter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>vuMeter</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PreviewWidget</class>
   <extends>QWidget</extends>
   <header>previewWidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>VuMeterWidget</class>
   <extends>QWidget</extends>
   <header>vuMeterWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>UltragridWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>556</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>arguments</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>682</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>683</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>682</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>606</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>682</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_3</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>682</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_5</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>682</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_7</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>682</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>682</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_6</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>682</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previewCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>previewLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previewCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>previewsSplitter</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
